crates/js-sys/tests/wasm/Temporal.js

```diff
@@ -0,0 +1,9035 @@
+exports.is_temporal_supported = function () {
```

Comment:
Is this a polyfill of `Temporal`? If so I think it's fine to leave this out and have the tests be opt-in and used only if `Temporal` actually exists (or disabled at compile-time by default or something like that)